tests/components/pterodactyl/conftest.py

```diff
@@ -17,6 +18,64 @@
     CONF_URL: TEST_URL,
     CONF_API_KEY: TEST_API_KEY,
 }
+TEST_SERVER_DATA_1 = {
+    "server_owner": True,
+    "identifier": "1",
+    "internal_id": 1,
+    "uuid": "1-1-1-1-1",
+    "name": "Test Server 1",
+    "node": "default_node",
+    "is_node_under_maintenance": False,
+    "sftp_details": {"ip": "192.168.0.1", "port": 2022},
+    "description": "",
+    "limits": {
+        "memory": 2048,
+        "swap": 1024,
+        "disk": 10240,
+        "io": 500,
+        "cpu": 100,
+        "threads": None,
+        "oom_disabled": True,
+    },
+    "invocation": "java -jar test1.jar",
+    "docker_image": "test_docker_image1",
+    "egg_features": ["eula", "java_version", "pid_limit"],
+    "feature_limits": {"databases": 0, "allocations": 0, "backups": 3},
+    "status": None,
+    "is_suspended": False,
+    "is_installing": False,
+    "is_transferring": False,
+    "relationships": {"allocations": {...}, "variables": {...}},
+}
+TEST_SERVER_DATA_2 = {
```

Comment:
Can we put this in a json file fixture?

Reply:
this wasnt put into a json file tho

Reply:
Here they are (?):

- tests/components/pterodactyl/fixtures/server_1_data.json
- tests/components/pterodactyl/fixtures/server_2_data.json

Reply:
Or did you mean a single JSON file for everything?

Reply:
Oh wait I misread the commits, youre right